src/sunburst.js

```diff
@@ -194,7 +196,7 @@ export default Kapsule({
 
     // Entering
     const newSlice = slice.enter().append('g')
-      .attr('class', 'slice')
+      .attr('class', (d) => 'slice ' + nodeCss(d.data))
```

Comment:
Please use string literals instead of string concat for building this attribute. Also, please check it's defined and not empty before adding unnecessary additional spaces to the class.

Reply:
Updated.